rest/permissions.py

```diff
@@ -114,3 +114,19 @@ def has_permission(self, request, view):
                 RepoPermission.manage_taxonomy[0]
                 in get_perms(request.user, repo)
             )
+
+
+class ManageRepoMembersPermission(BasePermission):
+    """Checks manage_repo_users permission"""
+
+    def has_permission(self, request, view):
+        if request.method in SAFE_METHODS:
+            return True
```

Comment:
[Same comment](https://github.com/mitodl/lore/pull/183#discussion_r32950841) as given previously in another PR.